lib/src/model/game/material_diff.dart

```diff
@@ -80,8 +104,16 @@ class MaterialDiff with _$MaterialDiff {
     });
 
     return MaterialDiff(
-      black: MaterialDiffSide(pieces: black.toIMap(), score: -score),
-      white: MaterialDiffSide(pieces: white.toIMap(), score: score),
+      black: MaterialDiffSide(
+        pieces: black.toIMap(),
+        score: -score,
+        capturedPieces: blackCapturedPieces,
```

Comment:
Can you extend `test/model/game/material_diff_test.dart` with a check for `capturedPieces`?